docs/content/docs/changelogs/v0.48.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
I was using the date for the file name earlier because I was anticipating that runtime package releases that don't have a corresponding 3P release might get some notes here as well (particularly to list exactly which packages got updated).  It certainly doesn't need to go here, but just a thought.

Reply:
Ahhh good point. We can add some aliases (redirects) in the future if we need to move things around.